src/Controls/tests/UITests/Tests/Issues/_IssuesUITest.cs

```diff
@@ -39,8 +39,7 @@ protected override void FixtureTeardown()
 			base.FixtureTeardown();
 			try
 			{
-				this.Back();
-				App.Click("GoBackToGalleriesButton");
+				Reset();
```

Comment:
We are now replacing the main page, it no longer makes sense to navigate back. For that reason, we do here a reset.

Reply:
Maybe for the first version here we don't reset the app

Just reset back to the root page here and then the next test will do a search 

<img width="327" alt="image" src="https://github.com/dotnet/maui/assets/5375137/437da88f-d588-49d0-9373-4889cf437c5d">

Reply:
Actually, if we reset the MainPage, we cannot navigate back. If we reset we have to reset, otherwise we can keep the navigation back.

Reply:
yea, you're right

I keep getting lost with the separation of runner and app under test :-)